libraries/src/Table/Extension.php

```diff
@@ -112,9 +113,41 @@ public function find($options = array())
 		// Get the DatabaseQuery object
 		$query = $this->_db->getQuery(true);
 
+		// Get the column field types for the #__extensions Table
+		$tableColumnsType = $this->_db->getTableColumns('#__extensions', false);
+
 		foreach ($options as $col => $val)
 		{
-			$query->where($col . ' = ' . $this->_db->quote($val));
+			if (\in_array($col, (array) $tableColumnsType[$col]))
+			{
+				// Mysql and Postgresql have different properties name for a column data
+				$type = isset($tableColumnsType[$col]->Type) ? $tableColumnsType[$col]->Type : $tableColumnsType[$col]->type;
```

Comment:
```suggestion
				$type = $tableColumnsType[$col]->Type ?? $tableColumnsType[$col]->type;
```

Beside this wouldn't it make more sense to fix this in our db layer?

Reply:
i've tryied  to be as more agnostic as i can , evidently i've failed
to me this is not the right place to deal with different  db's